doctr/travis.py

```diff
@@ -401,3 +389,22 @@ def push_docs(deploy_branch='gh-pages'):
     run(['git', 'pull', 'doctr_remote', deploy_branch])
     print("Pushing commit")
     run(['git', 'push', '-q', 'doctr_remote', deploy_branch])
+
+def determine_push_rights(branch_whitelist):
```

Comment:
yes, but let's do that separately -- I think this is good enough for now, since the alternative is broken functionality